docs/develop/debug-office-add-ins-in-visual-studio.md

```diff
@@ -152,6 +170,21 @@ Next, Visual Studio does the following:
 
 ### Debug the Outlook add-in
 
+The best method for debugging an add-in in Visual Studio 2022 depends on whether the add-in is running in WebView2, which is the webview control that is associated with Microsoft Edge (Chromium), or an older webview control. To determine which webview control is being used, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md). If your computer is using WebView2, see [Use the built-in Visual Studio debugger](#use-the-built-in-visual-studio-debugger). For any other webview control, see [Use the browser developer tools](#use-the-browser-developer-tools).
+
+#### Use the built-in Visual Studio debugger
+
+1. Set breakpoints, as needed, in the source JavaScript or TypeScript files. You can do this either before or after you start the add-in as described in the earlier section [Start the Excel, PowerPoint, or Word add-in project](#start-the-excel-powerpoint-or-word-add-in-project).
```

Comment:
if you accept my suggestion to uplevel/move/rename the "Start the Excel, PowerPoint, or Word add-in project" section, remember to update the title and anchor

Reply:
Seems like this should point to the Outlook version of that section...?
```suggestion
1. Set breakpoints, as needed, in the source JavaScript or TypeScript files. You can do this either before or after you start the add-in as described in the earlier section [Start the Outlook add-in project](#start-the-outlook-add-in-project).
```